Frontend/Program.cs

```diff
@@ -7,5 +7,5 @@
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
 builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
```

Comment:
This will map to "https://localhost:7206/" instead of "https://localhost:7038/api"
```suggestion
builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri("https://localhost:7038/api") });
```

Reply:
Should be managed via config file then (TODO)